components/brave_wallet/common/fil_address.cc

```diff
@@ -17,9 +17,9 @@ namespace brave_wallet {
 namespace {
 
 constexpr size_t kChecksumSize = 4;
-constexpr size_t kPublicKeySizeSecp256K = 20;
+constexpr size_t kHashLengthSecp256K = 20;
 constexpr size_t kAddressSizeSecp256K = 41;
-constexpr size_t kPublicKeySizeBLS = 48;
+constexpr size_t kHashLengthBLS = 48;
```

Comment:
this is actually the public key size, it's not hash.

Reply:
done